image-streams/tc510525.json

```diff
@@ -0,0 +1,21 @@
+{
+
+    "kind": "ImageStream",
+
+    "apiVersion": "v1",
+
+    "metadata": {
+
+      "name": "python",
```

Comment:
Your image is ruby,but your name is python,or you could use more meaningful name?